manufacturing-server/src/main.rs

```diff
@@ -165,21 +167,15 @@ impl TryFrom<DiunSettings> for DiunConfiguration {
     }
 }
 
-fn load_rendezvous_info(path: &str) -> Result<RendezvousInfo> {
-    let contents = match fs::read(path) {
-        Ok(c) => c,
-        // backwards compatibility for tests...
-        Err(_) => return Ok(RendezvousInfo::new(Vec::new())),
-    };
-    let mut info: Vec<RendezvousDirective> = Vec::new();
-
+fn load_rendezvous_info(path: &AbsolutePathBuf) -> Result<RendezvousInfo> {
     let value: Value =
-        serde_yaml::from_slice(&contents).context("Error parsing rendezvous info")?;
+        serde_yaml::from_slice(&fs::read(path)?).context("Error parsing rendezvous info")?;
```

Comment:
Let's do an `let contents = fs::read(path).context("Failed to read rendezvous info");`, and then pass `&contents` in here: basically make sure that the file read returns a different error string than failing to parse.

Reply:
fixed